backend/app/services/implementations/log_records_service.py

```diff
@@ -153,9 +155,15 @@ def get_log_records(self, page_number, filters=None):
             log_records = db.session.execute(text(sql))
             json_list = self.to_json_list(log_records)
             num_results = len(json_list)
+
+            # overwrite json_list to be the subset of data to return if return_all is false
+            if not return_all:
+                json_list = json_list[start_index:end_index]
+
             return {
-                "log_records": json_list[start_index:end_index],
+                "log_records": json_list,
                 "num_results": num_results,
+                "return_all": return_all,
```

Comment:
Is there a point of returning `return_all`

i.e. are we ever going to use it anywhere

Reply:
not really, think we left this in accidentally when we were testing